pkgs/top-level/all-packages.nix

```diff
@@ -420,6 +420,8 @@ in
 
   srcOnly = args: callPackage ../build-support/src-only args;
 
+  substitute = callPackage ../build-support/substitute/substitute.nix { };
```

Comment:
Why is this not using default.nix as the filename?

Reply:
ah, didn't catch that. I can just push a quick fix if you don't mind a direct push